src/main/java/com/rocketden/main/exception/UserErrors.java

```diff
@@ -0,0 +1,10 @@
+package com.rocketden.main.exception;
+
+import com.rocketden.main.exception.api.ApiError;
+import org.springframework.http.HttpStatus;
+
+public class UserErrors {
+
+    public final static ApiError INVALID_USER = new ApiError(HttpStatus.BAD_REQUEST,
+            "Please provide a user with a valid nickname (1-16 characters without spaces).");
```

Comment:
Related to my previous comments, alternatively we could move these error messages out to some sort of reference table or dictionary keyed with custom unique codes instead of having them all hard coded in various places. There's a couple of error messages on the front end files that could benefit from more organization too.

Reply:
I talked about this with Chris in an above comment where a HashMap was brought up (although not sure if that's what you were referring to when you said dictionary), but I think we went against it because that would mean extra overhead plus it would decentralize the errors (forcing the programmer to manually look up a message from the map when throwing an error). So I guess in a sense this file is our centralized "reference table" for errors, although there definitely could be a better way of doing it - did you have anything in mind for this, like Java enums or some other useful APIs?

Reply:
Maybe like moving the errors into a JSON or other data file of some sort? Especially as we start to get more and more errors, then maybe that might make it easier to add and edit them.